tests/src/Interop/StructMarshalling/PInvoke/MarshalStructAsParamDLL.cpp

```diff
@@ -1120,3 +1120,20 @@ extern "C" DLL_EXPORT BOOL STDMETHODCALLTYPE MarshalStructAsParam_AsExpByRefOutL
 	return TRUE;
 }
 
+extern "C" DLL_EXPORT HFA STDMETHODCALLTYPE GetHFA(float f1, float f2, float f3, float f4)
+{
+    return {f1, f2, f3, f4};
+}
+
+extern "C" DLL_EXPORT ManyInts STDMETHODCALLTYPE GetMultiplesOf(int value)
+{
+    int multiples[20];
+
+    for(int i = 0; i < 20; ++i)
+    {
+        multiples[i] = value * (i + 1);
+    }
+
+    ManyInts returnValue = *reinterpret_cast<ManyInts*>(&multiples);
```

Comment:
Wow. That is some very interesting code. Are you trying to hit _all_ the undefined behavior rules in the C++ spec? 😄 I realize it is more code and pedantic, but please set each field properly.

Reply:
Had a feeling you'd say that haha. I'll got change that to be more explicit.